purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesOrchestrator.kt

```diff
@@ -775,6 +780,10 @@ internal class PurchasesOrchestrator constructor(
     //endregion
 
     // region Private Methods
+    @Synchronized
```

Comment:
NABD but I think this doesn't need to be synchronized?

Reply:
Right! Not sure why I added this lol